src/Octoshift/Services/GithubApi.cs

```diff
@@ -800,13 +800,19 @@ public virtual async Task<ReattributeMannequinToUserResult> ReclaimMannequinsSki
             variables = new { orgId, sourceId = mannequinId, targetId = targetUserId }
         };
 
-        var response = await _client.PostAsync(url, payload);
-        var data = JObject.Parse(response);
-
-        return data["errors"].Any(x => ((string)x["message"]).Contains("Field 'reattributeMannequinToUser' doesn't exist on type 'Mutation'"))
-            ? throw new OctoshiftCliException("Feature Flag not enabled, ensure Feature Flag is enabled before re-attempting " +
-            "mannequin reattribution with --skip-invitation option")
-            : data.ToObject<ReattributeMannequinToUserResult>();
+        try
+        {
+            return await _retryPolicy.Retry(async () =>
+            {
+                var data = await _client.PostGraphQLAsync(url, payload);
+                return data.ToObject<ReattributeMannequinToUserResult>();
+            });
+        }
+        catch (Exception ex)
```

Comment:
this could be a catch OctoshiftCliException - which will also make CodeQL happy. You'll still want to filter it down to the specific message that indicates it was a FF flag problem.

```suggestion
        catch (OctoshiftCliException ex) when (ex.Message.Contains("Field 'reattributeMannequinToUser' doesn't exist on type 'Mutation'"))
```

Reply:
Updated, thanks!